pkg/aws/aws.go

```diff
@@ -76,15 +77,18 @@ func NewClient(accessID, accessSecret, token, region string) (AwsClient, error)
 
 // NewClientFrmFileCredentials creates a new client by reading credentials from a file
 func NewClientFrmFileCredentials(dir string, region string) (AwsClient, error) {
-	filepath := strings.TrimSuffix(dir, "/")
+	dir = strings.TrimSuffix(dir, "/")
+	dir = filepath.Clean(dir)
```

Comment:
can we not leave this one out, and only do it once in 83?

Reply:
and line 80 may also be unrequired, as the 'path.Join' should take care of it how to concatenate the path appropriately. But I will test it